hack/s3_test/run_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -ex
```

Comment:
This file looks unused now

Reply:
I'm using it to launch the test locally.

Reply:
If we have a helper script we should call it in the CI. Otherwise nobody notices if it breaks or starts to behave differently than CI invocations.

Reply:
Used it in the CI